data/json/vitamin.json

```diff
@@ -3,39 +3,64 @@
     "id": "calcium",
     "type": "vitamin",
     "name": "Calcium",
-    "min": -600,
-    "deficiency": [ [ "brittlebone", -300 ] ]
+    "deficiency": "hypocalcemia",
+    "min": -720,
+    "disease": [
+      [ -288, -336 ],
+      [ -337, -384 ],
+      [ -385, -720 ]
```

Comment:
> Then proper names for disease range IMHO would be disease_min and disease_max.

`min` and `max` are sufficient. There are values within that range that produce no diseases.

> You make an assumption that number of levels in vitamin and in effect for diseases are equal

Yes, it's reasonable to expect anyone implementing further vitamins to provide matching severity and diseases levels.

> Besides, it's inconvenient

I don't want to start performing micro-optimisations of the JSON until both the implementation is otherwise complete and there is evidence that we intend to add further vitamins.